src/client/order-components/GoogleAPI.js

```diff
@@ -2,7 +2,12 @@
 let map;
 const formatAnswer = (mode, value) => {
   if (!value) {
-    return 'not available';
+    // return 'not available';
+    if (mode === 'fast') {
+      return '5 minutes';
```

Comment:
what do we want the web client to display if a vendor doesn't return us results?
(improbably in production, but an edge case...) --> `N/A` ok ?